Dockerfile

```diff
@@ -1,15 +1,52 @@
-FROM ubuntu:focal
+FROM python:3.9-slim
+
+LABEL org.opencontainers.image.source="hhttps://github.com/UST-QuAntiL/qhana-plugin-runner"
```

Comment:
The url has one "h" too many.

Reply:
Should be fixed in all three repositories now.